collections/scheduledMessages.js

```diff
@@ -9,7 +9,7 @@ ScheduledMessages.attachSchema(new SimpleSchema({
   batchSize: {type: Number, optional: true},
   completed: {type: Boolean, defaultValue: false},
   context: {type: String, optional: true},
-  deliveryFailureUsers: {type: [String], blackbox: true, optional: true},
+  deliveryFailureUsers: {type: [String], defaultValue: []},
```

Comment:
Judging by the [https://github.com/aldeed/meteor-simple-schema/blob/master/DOCS.md](relevant meteor docs), it looks like blackbox isn't relevant here because it only applies to objects; is that right?

Reply:
Yes, that's right. That was left over from an earlier, ill-advised draft.